snowflake/DataIngestWithCopyTask.txt

```diff
@@ -0,0 +1,33 @@
+-- Create database --
+
+create database TEST_DB;
+
+create table "TEST_DB"."PUBLIC"."MYTABLE" ("LOANNUM" NUMBER (38,0), "BRANCHNAME" VARCHAR(50), "AMOUNT" NUMBER(20,2));
+
+use role accountadmin;
+ 
+-- Create stage -- 
+
+create or replace stage azure_stage
+  URL='azure://<storageaccount>.blob.core.windows.net/snowdata-blob'
+  CREDENTIALS=(AZURE_SAS_TOKEN='?<sastoken>');
+  
+-- Create task -- 
+ 
+create or replace task copy_data
+WAREHOUSE=PLAYWH
+SCHEDULE='1 MINUTE'
+AS
+copy into MYTABLE from @azure_stage
+FILE_FORMAT=(TYPE='CSV');
+    
+alter task copydata resume;
+ 
+-- Show results --
+ 
+select * FROM TEST_DB.PUBLIC.MYTABLE;
```

Comment:
Same as above, we can use _USE_ statement and execute just `select * FROM MYTABLE`

Reply:
Third thing, we during table creation we use notation with double quotes, but later without quotes. Let's be consistent.